### PR TITLE
Add MirrorConf website into showcase

### PR DESCRIPTION
**Pull request by elrumordelaluz**

### CODE REVIEW

File: docs/sites.yml [hunk 1]
```diff
@@ -2898,3 +2898,13 @@
   built_by: "https://fcfcu.com/"
   built_by_url: "https://fcfcu.com/"
   featured: false
+- title: Mirror Conf
+  description: >
+    Mirror Conf is a conference designed to empower designers and front-end developers who have a thirst for knowledge and want to broaden their horizons.
```
Comment (ahinrichs) [inline docs/sites.yml]: Remove the trailing space for lint to suceed

Comment (ahinrichs) [reply docs/sites.yml]: Sorry for not giving the field. Not the EOL at the end was wrong but the space at the end of the description field

Comment (elrumordelaluz) [reply docs/sites.yml]: np, so the EOL is needed, right?

Comment (ahinrichs) [reply docs/sites.yml]: yes

**Pull request merged**